src/app/components/identities/identities.constants.js

```diff
@@ -0,0 +1,6 @@
+  'use strict';
+
+  angular.module('esn-frontend-account-inbox')
```

Comment:
Can you change the name of module to `esn.account-inbox`?

_But still keep the SPA module name esn-frontend-account-inbox._